ERROR - 2023-03-27 03:51:29 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in F:\XAMPP\htdocs\suhu\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 F:\XAMPP\htdocs\suhu\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ews', 3306, '', 0)
#1 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\XAMPP\htdocs\suhu\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\XAMPP\htdocs\suhu\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 F:\XAMPP\htdocs\suhu\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 F:\XAMPP\htdocs\suhu\app\Controllers\Home.php(28): CodeIgniter\BaseModel->first()
#7 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 F:\XAMPP\htdocs\suhu\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 F:\XAMPP\htdocs\suhu\system\Commands\Server\rewrite.php(47): require_once('F:\\XAMPP\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in F:\XAMPP\htdocs\suhu\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\XAMPP\htdocs\suhu\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\XAMPP\htdocs\suhu\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 F:\XAMPP\htdocs\suhu\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 F:\XAMPP\htdocs\suhu\app\Controllers\Home.php(28): CodeIgniter\BaseModel->first()
#6 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\XAMPP\htdocs\suhu\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 F:\XAMPP\htdocs\suhu\system\Commands\Server\rewrite.php(47): require_once('F:\\XAMPP\\htdocs...')
#11 {main}
CRITICAL - 2023-03-27 03:51:29 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `ews_table`
WHERE `status` = :status:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Home.php(28): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-27 03:51:33 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in F:\XAMPP\htdocs\suhu\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 F:\XAMPP\htdocs\suhu\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'ews', 3306, '', 0)
#1 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\XAMPP\htdocs\suhu\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\XAMPP\htdocs\suhu\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 F:\XAMPP\htdocs\suhu\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 F:\XAMPP\htdocs\suhu\app\Controllers\Home.php(28): CodeIgniter\BaseModel->first()
#7 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 F:\XAMPP\htdocs\suhu\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 F:\XAMPP\htdocs\suhu\system\Commands\Server\rewrite.php(47): require_once('F:\\XAMPP\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in F:\XAMPP\htdocs\suhu\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\XAMPP\htdocs\suhu\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\XAMPP\htdocs\suhu\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\XAMPP\htdocs\suhu\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 F:\XAMPP\htdocs\suhu\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 F:\XAMPP\htdocs\suhu\app\Controllers\Home.php(28): CodeIgniter\BaseModel->first()
#6 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 F:\XAMPP\htdocs\suhu\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\XAMPP\htdocs\suhu\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 F:\XAMPP\htdocs\suhu\system\Commands\Server\rewrite.php(47): require_once('F:\\XAMPP\\htdocs...')
#11 {main}
CRITICAL - 2023-03-27 03:51:33 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `ews_table`
WHERE `status` = :status:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Home.php(28): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-27 08:03:44 --> Undefined variable $result
in APPPATH\Views\python.php on line 10.
 1 APPPATH\Views\python.php(10): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $result', 'APPPATH\\Views\\python.php', 10)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\python.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('python', [], true)
 5 APPPATH\Controllers\Python.php(19): view('python', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Python->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Python))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-27 08:04:06 --> Undefined variable $ac_tomorrow_norm
in APPPATH\Controllers\Python.php on line 18.
 1 APPPATH\Controllers\Python.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $ac_tomorrow_norm', 'APPPATH\\Controllers\\Python.php', 18)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Python->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Python))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-27 08:04:07 --> Undefined variable $ac_tomorrow_norm
in APPPATH\Controllers\Python.php on line 18.
 1 APPPATH\Controllers\Python.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $ac_tomorrow_norm', 'APPPATH\\Controllers\\Python.php', 18)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Python->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Python))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-27 08:05:08 --> Invalid file: "my_view.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('my_view.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('my_view', [], true)
 3 APPPATH\Controllers\Python.php(18): view('my_view', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Python->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Python))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
